demos/inverter-fleet/deployment/eu-west-prod-001/fleet.yaml

```diff
@@ -17,7 +17,13 @@ spec:
           gitRef:
             repository: default-repo
             targetRevision: main
```

Comment:
The example will not work unless it's merged to main.
To test it locally, change targetRevision: add-k8s-secret